lib/codecs/h264.js

```diff
@@ -150,6 +150,8 @@ PROFILES_WITH_OPTIONAL_SPS_DATA = {
   86: true,
   118: true,
   128: true,
+
+  // TODO: these three don't appear to be in the specificiation anymore?
```

Comment:
These don't appear in the spec, but I doubt there is any harm to keeping them here for now.

Reply:
which three? 138,139,134?

Reply:
yeah the three below don't appear at the lines specified in the specs that we have.

Reply:
updated the comment to clarify it a bit better.

Reply:
Just noticed that diff hid the next line that would've shown there isn't more. I guess I forgot to expand it and that made it a bit unclear. The new comment is nicer, though, thanks!